internal/rpc/block_reader.go

```diff
@@ -28,8 +28,7 @@ type BlockReader struct {
 	// UseDatanodeHostname specifies whether the datanodes should be connected to
 	// via their hostnames (if true) or IP addresses (if false).
 	UseDatanodeHostname bool
-	// DialFunc is used to connect to the datanodes. If nil, then
-	// (&net.Dialer{}).DialContext is used.
+	// DialFunc is used to connect to the datanodes. If nil, then (&net.Dialer{}).DialContext is used
```

Comment:
Please keep the newline (and period)

Reply:
funny story, my gocode linter actually says it *doesn't* want the punctuation at the end lol. I'll put it back though.